### PR TITLE
[CP] Add support for persona KYC

### PR DESCRIPTION
**Pull request by CapCap**
Create wrapper + add support for Persona KYC

This is for the happy path: user completes and returns. There are a few additional methods in the helper, this is for the unhappy path: we get updates via webhook because user may not have completed it interactively.

Unhappy path coming in a follow up PR because I am exhausted, and worst case we can manually run a sync job locally via chron a few times for a day or two until it's fully automated.

### CODE REVIEW

File: ecosystem/platform/server/app/views/welcome/it1.html.erb [hunk 1]
```diff
@@ -30,9 +32,13 @@
...
           <hr class="border-[#1dd7a5] mb-4">
           <p class="mb-8">
-          Identity must be verified by noon (PST) May 23 to get into the lottery.
+          Identity must be verified by noon (PST) May 23 to get into the selection process.
           </p>
-          <%= link_to "Verify", "#", class: "w-full bg-[#1dd7a5] text-neutral-800 block p-2 text-center font-mono uppercase text-lg rounded font-bold mt-auto" %>
+          <% if current_user&.it1_profile&.validator_verified? %>
+            <%= link_to "Verify", onboarding_kyc_redirect_path, class: "w-full bg-[#1dd7a5] text-neutral-800 block p-2 text-center font-mono uppercase text-lg rounded font-bold mt-auto" %>
+          <% else %>
+            <%= link_to "Verify", "#", class: "w-full bg-[#1dd7a5] text-neutral-800 block p-2 text-center font-mono uppercase text-lg rounded font-bold mt-auto" %>
```
Comment (zacharydenton) [inline ecosystem/platform/server/app/views/welcome/it1.html.erb:40]: If not verified, should the link appear at all?

File: ecosystem/platform/server/app/controllers/health_controller.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+# frozen_string_literal: true
+
+class HealthController < ApplicationController
+  def health
+    User.where(id: 1).exists?
+    render plain: '🔥🤼🔥'
```
Comment (zacharydenton) [inline ecosystem/platform/server/app/controllers/health_controller.rb:8]: ??? I don't understand this

File: ecosystem/platform/server/config/routes.rb [hunk 3]
```diff
@@ -20,8 +20,14 @@
     resources :users, only: %i[show update]
   end
 
+  # KYC routes
+  get 'onboarding/kyc_redirect', to: 'onboarding#kyc_redirect'
+  get 'onboarding/kyc_callback', to: 'onboarding#kyc_callback'
+
   get 'onboarding/email', to: 'onboarding#email'
-  post 'onboarding/email', to: 'onboarding#email_update'
+  post 'onboarding/email', to: 'onboarding#kyc_callback'
```
Comment (zacharydenton) [inline ecosystem/platform/server/config/routes.rb:28]: Can you explain this change? I don't understand at first glance.

Comment (CapCap) [reply ecosystem/platform/server/config/routes.rb]: typo, fixed!

File: ecosystem/platform/server/app/controllers/health_controller.rb (hunk 2 above)
Comment (CapCap) [reply ecosystem/platform/server/app/controllers/health_controller.rb:8]: health endpoint; verifies DB connect works and server is serving. Will be called by AWS; if it fails, container gets relaunched.

If you're wondering why it's two people wrestling surrounded by flames, I figured page needs to return something, lets amuse anyone who tries calling `/health` on random servers to see what happens :-P

**Review by zacharydenton — APPROVED** [review]
LGTM

**Pull request merged**